nucypher/cli/main.py

```diff
@@ -140,15 +143,16 @@ def status(click_config, config_file):
 @click.argument('action')
 @click.option('--debug', '-D', help="Enable debugging mode", is_flag=True)
 @click.option('--dev', '-d', help="Enable development mode", is_flag=True)
-@click.option('--dry-run', help="Execute normally without actually starting the node", is_flag=True)
+@click.option('--quiet', '-Q', help="Disable logging", is_flag=True)
```

Comment:
heh

Reply:
I think this is too terse. How about: "Disable terminal echoing and logging"  or "We'll do it live" :joy:

Reply:
Haha, the former is nice. :p